demisto_sdk/commands/validate/tools.py

```diff
@@ -172,3 +172,14 @@ def is_indicator_pb(playbook: Playbook):
         (i.get("playbookInputQuery") or {}).get("queryEntity") == "indicators"
         for i in playbook.data.get("inputs", {})
     )
+
+
+def filter_nones(ls: Union[List, Tuple]) -> List:
```

Comment:
looks like this filters anything falsey no? lets rename if thats the case
is there a util file or something thats more general to the sdk?